README.adoc

```diff
@@ -384,51 +367,63 @@ include::{common-includes}/mvncompile.adoc[]
 
 In addition, a few endpoint tests have been provided for you to test the basic functionality of the `inventory` and `system` services. If a test failure occurs, then you must have introduced a bug into the code.
-Reminder that you must register the custom configuration source and custom converter in the `src/main/resources/META-INF/services/` directory, if you miss to complete these steps, the tests will fail.
+Reminder that you must register the custom configuration source and custom converter in the `src/main/resources/META-INF/services/` directory, if you miss to complete these steps, the tests will fail. These tests
+will run automatically as a part of the integration test suite.
+
 
 [role=command]
-include::{common-includes}/mvnverify.adoc[]
+include::{common-includes}/devmode-test.adoc[]
+
+You will see the following output:
 
 [source, role="no_copy"]
 ----
 -------------------------------------------------------
- T E S T S
--------------------------------------------------------
-Running it.io.openliberty.guides.config.ConfigurationTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.729 sec - in it.io.openliberty.guides.config.ConfigurationTest
-Running it.io.openliberty.guides.inventory.InventoryEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.477 sec - in it.io.openliberty.guides.inventory.InventoryEndpointTest
-Running it.io.openliberty.guides.system.SystemEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.013 sec - in it.io.openliberty.guides.system.SystemEndpointTest
-
-Results :
-
-Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
-
-----
-
-To see whether the tests detect a failure, remove the configuration resetting line in the [hotspot=setup file=0]`setup()` method of the [hotspot file=0]`ConfigurationTest.java` file.
+[INFO]  T E S T S
+[INFO] -------------------------------------------------------
+[INFO] Running it.io.openliberty.guides.config.ConfigurationIT
+[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.92 s - in it.io.openliberty.guides.config.ConfigurationIT
+[INFO] Running it.io.openliberty.guides.system.SystemEndpointIT
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.017 s - in it.io.openliberty.guides.system.SystemEndpointIT
+[INFO] Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
+[INFO]
+[INFO] Results:
+[INFO]
+[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0
+----
+
+To see whether the tests detect a failure, remove the configuration resetting line in the [hotspot=setup file=0]`setup()` method of the [hotspot file=0]`ConfigurationIT.java` file.
 Then manually change some configuration values in the `resources/CustomConfigSource.json` file.
```

Comment:
Change
"Then manually change some configuration values in the `resources/CustomConfigSource.json` file."
To
"Then, manually change some configuration values in the `resources/CustomConfigSource.json` file."

Reply:
Updated